cmake/projects/Boost/scripts/patched_boostrap.bat.in

```diff
@@ -1,3 +1,3 @@
 call "@HUNTER_MSVC_VCVARSALL@" @HUNTER_MSVC_ARCH@
-cd @HUNTER_PACKAGE_SOURCE_DIR@
+cd /d @HUNTER_PACKAGE_SOURCE_DIR@
```

Comment:
Please add comment with description of problematic case. Plus would be nice to have link to documentation.

Reply:
> Please update all PACKAGE_INTERNAL_DEPS_ID

Do we want to force an update of the cached binaries? This should produce identical binaries for builds on the C drive.

> Plus would be nice to have link to documentation.

Documentation for cd? https://technet.microsoft.com/en-gb/library/bb490875.aspx

Reply:
> Do we want to force an update of the cached binaries? This should produce identical binaries for builds on the C drive.

The rule of thumb is to change `PACKAGE_INTERNAL_DEPS_ID` every time build instructions changed. In this particular case if I will start tests now everything will be fetched from server and no build from source will be triggered. Effectively I will test nothing.

> Documentation for cd?

Yes, like "We have to use ... because if ... . Documentation: ..."